ci/repo_cleanups/Jenkinsfile.garbage_collection

```diff
@@ -0,0 +1,26 @@
+#!groovy
+
+import org.jenkinsci.plugins.workflow.steps.FlowInterruptedException
+// http://stackoverflow.com/questions/37425064/how-to-use-environment-variables-in-a-groovy-function-using-a-jenkinsfile
+import groovy.transform.Field
+
+def checkout_and_merge() {
+    checkout scm
+    sh "git merge origin/master"
+}
+
+def stage_clean_rpms() {
+  node() {
+    stage "RPM Repository Cleanups"
+    checkout scm
+    sh "git remote prune origin"
+    sh "cd ci/repo_cleanups ; ./remove_stale_rpm.sh"
+  }
+}
+
+node() {
+    stage "Checkout"
+//  checkout_and_merge()
```

Comment:
I think it's better to remove commented out code before merging. Git makes it possible to get old code back and I think it makes the code cleaner.